motor.h

```diff
@@ -1,5 +1,5 @@
-#ifndef Motor_h
-#define Motor_h
+#ifndef motor_h
```

Comment:
Please also put comment on line 15 in this way:
```cpp
#endif  // MOTOR_H
```
Note the 2 spaces between `#endif` and `//`.